packages/strapi-hook-bookshelf/lib/index.js

```diff
@@ -89,7 +89,7 @@ module.exports = function(strapi) {
             // Register the final model for Bookshelf.
             const loadedModel = _.assign({
               tableName: definition.collectionName,
-              hasTimestamps: _.get(definition, 'options.timestamps') === true,
```

Comment:
This will be a breaking change, if you were migrate with the older default values `0 or 1` I'm pretty sure that Strapi will throw an error. The timestamp names I think should have it's own option, if no value is provided then use the defaults.

Reply:
Hey thanks for a quick response. Let me quickly change that :)

Reply:
I've added additional checks to take care of existing attribute values.